rkt/config/auth.go

```diff
@@ -59,7 +75,7 @@ type basicAuthHeaderer struct {
 	auth basicV1
 }
 
-func (h *basicAuthHeaderer) Header() http.Header {
+func (h *basicAuthHeaderer) GetHeader() http.Header {
```

Comment:
It's not really a getter, it simply has a proper method name (verb and everything). happy to change it to 'Header' again, but I find that quite misleading - 'Header' would be the name of a getter, and it just isn't a getter.